internal/mining/mining.go

```diff
@@ -154,17 +233,13 @@ const (
 // transaction to be prioritized and track dependencies on other transactions
 // which have not been mined into a block yet.
 type txPrioItem struct {
-	tx       *dcrutil.Tx
-	txType   stake.TxType
-	fee      int64
-	priority float64
-	feePerKB float64
-
-	// dependsOn holds a map of transaction hashes which this one depends
-	// on.  It will only be set when the transaction references other
-	// transactions in the source pool and hence must come after them in
-	// a block.
-	dependsOn map[chainhash.Hash]struct{}
+	tx          *dcrutil.Tx
+	txDesc      *TxDesc
+	txType      stake.TxType
```

Comment:
Do you think it makes sense to remove `tx` and `txType` since you are now storing `txType`, and they can be accessed through that?

Reply:
I think you mean since it's now storing `txDesc`, but I agree.  It's double storing them.  The same is true of the `fee` field.

Reply:
Removed redundant reference